Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -478,45 +478,15 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
 
             Task {
                 await self.handleAdapterStopped()
-
-                switch reason {
-                case .userInitiated:
-                    // stop requested by user from System Settings
-                    // we can‘t prevent a respawn with on-demand rule ON
-                    // request the main app to reconfigure with on-demand OFF
-
-                    await self.appLauncher?.launchApp(withCommand: .stopVPN)
-
-                case .superceded:
+                if case .superceded = reason {
                     self.notificationsPresenter.showSupersededNotification()
-
-                default:
-                    break
                 }
 
                 completionHandler()
-
-                #if os(macOS)
-                // From what I'm seeing in my tests the next call to start the tunnel is MUCH
-                // less likely to fail if we force this extension to exit when the tunnel is killed.
-                //
-                // Ref: https://app.asana.com/0/72649045549333/1204668639086684/f
-                //
-                exit(EXIT_SUCCESS)
-                #endif
```

Comment:
Could you expand on why this is being removed?

Reply:
same here, moved to macOS repo

Reply:
Could you add a link to the lines that replace this?  It's being really hard for me to follow the changes in these PRs as they touch a lot of different things.

Reply:
https://github.com/duckduckgo/macos-browser/blob/b5ed7355d48ac2997bf7f84b8f7fede2063cc755/DuckDuckGo/NetworkProtection/NetworkExtensionTargets/NetworkExtensionTargets/MacPacketTunnelProvider.swift#L269

https://github.com/duckduckgo/macos-browser/blob/b5ed7355d48ac2997bf7f84b8f7fede2063cc755/DuckDuckGo/NetworkProtection/NetworkExtensionTargets/NetworkExtensionTargets/MacPacketTunnelProvider.swift#L299